provider/v1alpha1/service.proto

```diff
@@ -59,6 +61,23 @@ message MountRequest {
 message MountResponse {
     repeated ObjectVersion object_version = 1;
     Error error = 2;
+    // files contains the entire mount volume filesystem.
+    repeated File files = 3;
+}
+
+// File holds secret file contents and location in the mount path to write the
+// file.
+message File {
+    // The relative path of the file within the mount.
+    // May not be an absolute path.
+    // May not contain the path element '..'.
+    // May not start with the string '..'.
+    string path = 1;
+    // The mode bits used to set permissions on this file.
+    // Must be a decimal value between 0 and 511.
+    int32 mode = 2;
```

Comment:
Today, the mode is sent to the provider from the driver. Having this will make it extensible in the future if we decide to support user configuration for mode.